test/Core.Test/AutoFixture/GlobalSettingsFixtures.cs

```diff
@@ -23,7 +23,9 @@ public object Create(object request, ISpecimenContext context)
             }
 
             var pi = request as ParameterInfo;
-            if (pi == null || pi.ParameterType != typeof(GlobalSettings))
+            var fixture = new Fixture();
+
+            if (pi == null || pi.ParameterType != typeof(Settings.GlobalSettings))
```

Comment:
`pi` can never be `null`, it's always `3.141592653589793238462643383279502884197169399375105820974944592307816406286` (just kidding, kind of).